### ISSUE TITLE
[nodejs] errors with mount-plain webpack bundled js file

### ISSUE THREAD

**Issue by moritzraho** [adobe/wskdebug#24]
## Description
wskdebug for `mount-plain.js` (used in not zipped actions) is failing for webpack bundled actions while the same bundled js file works with `mount-require.js` (used in zipped action)

## Error

`> wskdebug my-bundled-action path/to/myaction.js`

`#do the invocation`

```
Activation: 15632c7e0da648a0a32c7e0da6c8a022
2019-11-07T09:18:34.487541760Z RangeError: Maximum call stack size exceeded
2019-11-07T09:18:34.487609654Z     at Object.eval (/code/index.js:97:16)
2019-11-07T09:18:34.487619477Z     at __webpack_require__ (/code/index.js:21:30)
2019-11-07T09:18:34.487623701Z     at Object.module.exports.exports.main (/code/index.js:92:18)
2019-11-07T09:18:34.487627441Z     at __webpack_require__ (/code/index.js:21:30)
2019-11-07T09:18:34.487631131Z     at eval (/code/index.js:85:18)
2019-11-07T09:18:34.487634802Z     at eval (/code/index.js:88:10)
2019-11-07T09:18:34.487638372Z     at eval (/code/index.js:140:16)
2019-11-07T09:18:34.487642048Z     at load (eval at NodeActionRunner.init (/nodejsAction/runner.js:80:45), <anonymous>:30:16)
2019-11-07T09:18:34.487646464Z     at main (eval at NodeActionRunner.init (/nodejsAction/runner.js:80:45), <anonymous>:45:24)
2019-11-07T09:18:34.487650648Z     at main (eval at NodeActionRunner.init (/nodejsAction/runner.js:80:45), <anonymous>:48:12)
2019-11-07T09:18:34.487856218Z XXX_THE_END_OF_A_WHISK_ACTIVATION_XXX
2019-11-07T09:18:34.488231343Z XXX_THE_END_OF_A_WHISK_ACTIVATION_XXX
Completed activation 15632c7e0da648a0a32c7e0da6c8a022 in 18.025 sec
```

The problem seems to come from nested evals

## Webpack config
```
entry: [
            actionPath
          ],
          output: {
            path: buildDir,
            filename: buildFilename,
            libraryTarget: 'commonjs2'
          },
          // see https://webpack.js.org/configuration/mode/
          mode: 'production',
          target: 'node',
          optimization: {
            // error on minification for some libraries
            minimize: false
          },
          // the following lines are used to require es6 module, e.g.node-fetch which is used by azure sdk
          resolve: {
            extensions: ['.js'],
            mainFields: ['main']
          },
```

## Workaround

- rename `path/to/myaction.js` to `index.js`
- zip it
- update your action with the new zip action
=> wskdebug works fine

**Comment by alexkli** [comment]
This might be fixed by #27.

**Comment by moritzraho** [comment]
👍 seems to fix it
